### PR TITLE
Fix for Timestamp with Time Zone datatype - #6

### PR DESCRIPTION
**Pull request by tbrugz**
*Issue #, if available:* #6 
 
*Description of changes:*
Parsing time zone information from timestamp inside Strings, tests also added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/main/java/com/amazon/opendistroforelasticsearch/jdbc/types/TimestampType.java [hunk 1]
```diff
@@ -73,6 +74,19 @@ private Timestamp localDateTimeToTimestamp(LocalDateTime ldt, Calendar calendar)
             if (value.length() > 11 && value.charAt(10) == 'T') {
                 value = value.replace('T', ' ');
             }
+            // Timestamp.valueOf() does not like timezone information
+            if (value.length() > 23) {
+                if(value.length() == 24 && value.charAt(23) == 'Z') {
```
Comment (qreshi) [inline src/main/java/com/amazon/opendistroforelasticsearch/jdbc/types/TimestampType.java]: NP: Please add a space after the ```if/else if``` statements on lines 79, 82 and 84.

File: src/main/java/com/amazon/opendistroforelasticsearch/jdbc/types/TimestampType.java [hunk 2]
```diff
@@ -73,6 +74,19 @@ private Timestamp localDateTimeToTimestamp(LocalDateTime ldt, Calendar calendar)
...
+                    value = value.substring(0, 23);
+                }
+                else if(value.charAt(23) == '+' || value.charAt(23) == '-') {
+                    // 'calendar' parameter takes precedence
+                    if(calendar==null) {
```
Comment (qreshi) [inline src/main/java/com/amazon/opendistroforelasticsearch/jdbc/types/TimestampType.java]: NP: Please also add a space around the ```==``` here and around the ```+```  for ```"GMT"+value.substring(23))``` for style consistency.

**Review by qreshi — CHANGES_REQUESTED** [review]

File: src/main/java/com/amazon/opendistroforelasticsearch/jdbc/types/TimestampType.java (hunk 2 above)
Comment (tbrugz) [reply src/main/java/com/amazon/opendistroforelasticsearch/jdbc/types/TimestampType.java]: ok, will do

**Review by qreshi — APPROVED** [review]
Looks good, thanks for the changes!

**Pull request merged**